proposals/exception-handling/Exceptions.md

```diff
@@ -311,6 +310,39 @@ catching instructions (`catch`/`catch_all`/`delegate`) come after the
 same as if the `try` has catches but none of the catches are able to handle the
 exception.
 
+If `delegate`'s immediate argument is the depth of block-like structures
+(blocks, loops, and trys) + 1, i.e., the function-level block, it is considered
```

Comment:
I don't understand what you mean by: "the depth of block-like structures (...) + 1" and how this is related to the function-level block case. Or did you mean that in this case the depth is *not* a block-like structure?

Reply:
I understand what you meant now. What do you think about this phrasing: "the depth of the outermost block + 1, i.e. the function-level block (...)"

Reply:
Done. Also added that it will be considered as a catchless try.